kofta/src/vscode-webview/pages/BanUsersPage.tsx

```diff
@@ -24,37 +24,39 @@ export const BanUsersPage: React.FC<SearchUsersProps> = ({}) => {
   }
 
   return (
-    <Wrapper>
+    <Wrapper style={{ marginBottom: "auto" }}>
```

Comment:
instead of putting this on every wrapper, should we just remove the padding from the wrapper component?

Reply:
Agreed.

Reply:
I think that would be a good idea. I get why the padding is there, but the backbar is inside the wrapper on every page, and applying padding padding to a backbar doesn't look good. This method only really works on the home and login page, because there is no backbar there.

Reply:
Ok then remove the padding from `<Wrapper />` and create a  `<BodyWrapper />` with the padding and use that on the main content for all the pages